build_scripts/get_version.py

```diff
@@ -1,12 +1,25 @@
 """Get the version of the package."""
 
 import importlib.metadata
+import requests
+import json
+
+APP_VERSION_URL = (
+    "https://api.github.com/repos/OpenAdaptAI/OpenAdapt/releases?per_page=10&page=1"
+)
 
 
 def get_version() -> str:
     """Get the version of the package."""
     return importlib.metadata.version("openadapt")
 
 
+def get_latest_version() -> str:
+    """Get the latest version of the app available."""
+    response = requests.get(APP_VERSION_URL, stream=True)
+    data = json.loads(response.text)
+    return data[0]["tag_name"].replace("v", "")
```

Comment:
What do you think about extracting this from `pyproject.toml` instead? e.g.

```
def get_latest_version() -> str:
    """Get the latest version of the app available."""
    toml_path = Path(config.PARENT_DIR_PATH) / "pyproject.toml"
    parser = TomlFileParser(str(toml_path))
    parsed_data = parser.parse()
    latest_version = parsed_data['tool']['poetry']['version']
    return latest_version
```

Reply:
No @abrichr.
That only works in local development as we take a gitpull and pyproject.toml would get updated. 
But in case of users that won't happen and this github version would be our source of truth.